.ci/scripts/install-dependencies.sh

```diff
@@ -17,10 +17,10 @@ GOPROXY_CONTEXT=${GOPROXY}
 GOPROXY=''
 
 # Install some other dependencies required for the pre-commit
-go get -v -u golang.org/x/lint/golint
-go get -v -u github.com/golangci/golangci-lint/cmd/golangci-lint@v1.18.0
-go get -v -u github.com/go-lintpack/lintpack/...
-go get -v -u github.com/go-critic/go-critic/...
+go get -v golang.org/x/lint/golint
+go get -v github.com/golangci/golangci-lint/cmd/golangci-lint@v1.18.0
+go get -v github.com/go-lintpack/lintpack/...
+go get -v github.com/go-critic/go-critic/...
```

Comment:
This can still connect to network if these dependencies are not already in the go path, are we copying them by other means?

Reply:
No, the flag will cause an update over network if needed. I checked that the flag causes the build to fail because of `go-critic`